code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -405,15 +397,15 @@
 	private MasterFacet masterFacet = FacetLibrary.getFacet(MasterFacet.class);
 	private AutoEquipmentListFacet autoListEquipmentFacet = FacetLibrary.getFacet(AutoEquipmentListFacet.class);
 	private FollowerFacet followerFacet = FacetLibrary.getFacet(FollowerFacet.class);
-	
+
```

Comment:
agreed. This is entirely my IDE doing this change. We should also pick a standard and stick to it.